src/nls/sr/strings.js

```diff
@@ -457,3 +518,5 @@ define({
     // extensions/default/WebPlatformDocs
     "DOCS_MORE_LINK"                            : "Прочитај више"
 });
+
+/* Last translated for 752856d58d2e9dde14e1af6be615bb7080727b7a */ 
```

Comment:
Where do you get this? This is not a valid SHA. You have to look up this [history page](https://github.com/adobe/brackets/commits/c292e896761bc7d451a9e3b95bedd20d6b355d77/src/nls/root/strings.js) and copy the SHA from the one that you're translating for. Make sure you have the exact total line numbers in your localized file when comparing (diffing) to the English strings and also ensure that your localized file does not have keys in a different order from `root/strings.js`.